lib/register_screen.dart

```diff
@@ -0,0 +1,179 @@
+              padding: const EdgeInsets.all(8.0),
+              child: TextField(
+                controller: uncontroller,
+                decoration: InputDecoration(
+                  fillColor: const Color(0xffEEEEFE),
+                  filled: true,
+                  helperText: "Username",
+                  hintText: "Username",
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                ),
+              ),
+            ),
+
+            //Email butonunu burada tanımladım.
+
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: TextField(
+                controller: eccontroller,
+                decoration: InputDecoration(
+                  fillColor: const Color(0xffEEEEFE),
+                  filled: true,
+                  helperText: "E-Mail",
+                  hintText: "example@example.com",
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                ),
+              ),
+            ),
+
+            //Password butonunu burada tanımladım.
+
+            Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: TextField(
+                controller: pccontroller,
+                decoration: InputDecoration(
+                  fillColor: const Color(0xffEEEEFE),
+                  filled: true,
+                  helperText: "Password",
+                  hintText: "Password",
+                  border: OutlineInputBorder(
+                    borderRadius: BorderRadius.circular(10),
+                  ),
+                ),
+              ),
+            ),
+
+            //Or register with butonunu burada tanımladım,altında sosyal medya logoları da bulunuyor.
+
+            const Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Text(
+                  "Or Register With",
+                  style: TextStyle(fontSize: 20),
+                ),
+              ],
+            ),
+            const SizedBox(
+              height: 20,
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Image.asset("images/ic_sc.png"),
+              ],
+            ),
+            const SizedBox(
+              height: 20,
+            ),
+
+            //Sosyal medya altındaki yazıyı burda tanımladım.
+
+            const Padding(
+              padding: EdgeInsets.all(8.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  Text(
+                    "All ready have an account  ?",
+                    style: TextStyle(fontSize: 15,
+                        color: Color(0xff191849),
+                        fontWeight: FontWeight.bold),
+                  ),
```

Comment:
Yazı fontları için bir paketimiz var onu seninle paylaşıncaya dek figmadaki boyutların aynısını yazmalısın henüz karşılaştıramadım ama eğer yazı boyutlarını rastgele girdiysen bunları tüm projede düzenlemelisin.